README.md

```diff
@@ -1,6 +1,8 @@
 # Swapify
 
-[![Build Status](https://travis-ci.org/fri-team/Swapify.svg?branch=master)](https://travis-ci.org/fri-team/Swapify)
+
+[![Build Status](https://github.com/fri-team/Swapify/workflows/GitHub%20Actions/badge.svg?branch=develop)](https://github.com/fri-team/Swapify/actions?query=branch%3Adevelop)
+<!--outdated travis[![Build Status](https://travis-ci.org/fri-team/Swapify.svg?branch=master)](https://travis-ci.org/fri-team/Swapify)-->
```

Comment:
You don't have to comment old code, we have git for history of changes

Reply:
Yes, you are right. Will be cleared later when we will update readme file in the future.